odoo/addons/base/models/ir_filters.py

```diff
@@ -25,6 +25,7 @@ class IrFilters(models.Model):
                                      "When left empty the filter applies to all menus "
                                      "for this model.")
     active = fields.Boolean(default=True)
+    topbar_action_id = fields.Many2one('ir.actions.topbar', string="The topbar action this filter is applied to", ondelete="cascade")
```

Comment:
Don't we have to add `export_string_translation=False` to every technical fields? :vomiting_face:

Reply:
?

Reply:
I asked Xavier because I didn't understand it very well, and from his pov, it is not necessary. But could elaborate why It should be necessary, I am not sure I get why we wouldn't want that label to not be translated ?